packages/client-python/src/rocketride/mixins/execution.py

```
@@ -254,14 +254,17 @@ async def use(
         # Load pipeline configuration from file if needed
         if not pipeline:
-            with open(filepath, 'r', encoding='utf-8') as file:
-                # Prefer JSON5 for better developer experience (comments, trailing commas)
-                if json5:
-                    parsed = json5.load(file)
-                else:
-                    parsed = json.load(file)
-                # .pipe files wrap the config in { "pipeline": { ... } } — unwrap if present
-                pipeline_config = parsed.get('pipeline', parsed) if isinstance(parsed, dict) else parsed
+            try:
+                with open(filepath, 'r', encoding='utf-8') as file:
```

Ruff (0.15.7)

[warning] Async functions should not open files with blocking methods like `open`

(ASYNC230)